packages/pxweb2/src/app/util/startPageFilters.ts

```diff
@@ -114,20 +114,46 @@ export function updateSubjectTreeCounts(
   return originalTree.map(updateNode);
 }
 
-export function sortAndDeduplicateFilterChips(filters: Filter[]): Filter[] {
-  const typeOrder = ['subject', 'timeUnit'];
-  const sorted = filters.toSorted((a, b) => {
+export function sortFiltersByTypeAndSubjectOrder(
+  filters: Filter[],
+  subjectOrder: string[],
+): Filter[] {
+  const typeOrder = ['subject', 'yearRange', 'timeUnit'];
+
+  return filters.slice().sort((a, b) => {
```

Comment:
Er det noen grunn til at du bruker array.slice().sort() istedenfor array.toSorted()?

Reply:
Ah, ja det er et nytt tilskudd, det kom kanskje i es2023. Da gjør vi det sånn for å beholde kompatibilitet :)